.python-version

```diff
@@ -0,0 +1,5 @@
+3.8.18
```

Comment:
is there a need to mention the micro version? couldn' t it be just 3.X instead of 3.X.Y?

Reply:
I found out that `pyenv` can auto-resolve the latest versions of each python version, see https://github.com/pyenv/pyenv#prefix-auto-resolution-to-the-latest-version. Indeed, it will no longer be necessary to specify the minor version.